### PR TITLE
ISSD-4058 minor corrections

### PR DESCRIPTION
**Pull request by rtienpal**

### CODE REVIEW

**Comment by rtienpal** [comment]
ci:test:sf

**Comment by rtienpal** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -257,7 +257,7 @@ pre[class*='language-'] {
 				padding: 0 4px;
 			}
 		}
-
+
```
Comment (ryanschuhler) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css]: @rtienpal we should be careful about whitespace, we dont want to add this here. Also make sure to run both gw formatSource before sending

**Comment by ryanschuhler** [comment]
@rtienpal when sending pulls make sure not to include any old commits that have already been sent but are not in master. You can bring them into your branch for development, but when sending them up we should remove them

**Comment by rtienpal** [comment]
ci:test:sf

**Comment by rtienpal** [comment]
Closing, please check https://github.com/liferay-customer-delivery/liferay-portal/pull/79

**Pull request closed without merging**